logic/pulsed/pulsed_measurement_logic.py

```diff
@@ -1153,6 +1159,7 @@ def _extract_laser_pulses(self):
         self.raw_data = fc_data
         self.__elapsed_sweeps = info_dict['elapsed_sweeps']
         self.__elapsed_time = info_dict['elapsed_time']
+        self.__trigger_ratio = (self.__elapsed_sweeps * self.__fast_counter_record_length) / self.__elapsed_time
```

Comment:
All the values to calculate this trigger ratio are already known to the fast counting hardware. So at least regarding calculating this ratio there is no need to involve the measurement logic.

Reply:
This is very true, I did not know most other hardware did not have a problem of dead time when I wrote this, which is my a approached it in a hardware agnostic perspective. If they don't then it's true this is quite useless !

To answer your previous message, the ratio does not fluctuate in time, because it's not a problem of buggy hardware.

The error I'm trying to prevent is when the user forget the dead time of the fast counter and send the next trigger too soon in the programmed sequence. If he does, he will have a constant 50% ratio, and this might help it prevent such error. That is what this PR is about. 

Another solution is we drop the PR and too bad for the PhD students who forget about the dead time !